analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DateTimeFormatShouldNotBeHardcoded.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Globalization;
+
+public class DateTimeFormatShouldNotBeHardcoded
+{
+    private string Format = $"dd/MM";
+
+    public void DateTimeCases()
+    {
+        var stringRepresentation = DateTime.UtcNow.ToString("dd/MM/yyyy HH:mm:ss"); // Noncompliant {{Do not hardcode the format specifier.}}
+//                                                 ^^^^^^^^
```

Comment:
I would prefer the location to be the hardcoded string instead of the method itself, but I get it might complicate things on the implementation side so it's fine

Reply:
No problem to change that. I will do it.